__tests__/config/formats/javascript-commonJs.test.ts

```diff
@@ -0,0 +1,63 @@
+import {javascriptCommonJs} from '../../../config/formats/javascript-commonJs'
+
+describe('Format: CommonJs', () => {
+  const dictionary = {
+    tokens: {
+      colors: {
+        red: {
+          $type: 'color',
+          value: '#FF0000'
+        }
+      }
+    }
+  }
+
+  const platform = {
+    prefix: 'test'
+  }
+
+  const file = {
+    destination: 'tokens.ts',
+    options: {
+      showFileHeader: false
+    }
+  }
+
+  it('Formats tokens adding prefix', () => {
+    const output = `exports.default = {
+  test: {
+    colors: {
+      red: "#FF0000",
+    },
+  },
+};
+`
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore: fake values to test formatter
+    expect(javascriptCommonJs({dictionary, file, options: undefined, platform})).toStrictEqual(output)
+  })
+
+  it('Formats tokens without prefix', () => {
+    const output = `exports.default = {
+  colors: {
+    red: "#FF0000",
+  },
+};
+`
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore: fake values to test formatter
```

Comment:
Worth stubbing these out? Could make them available globally as we seem to be using these in a few different tests.

Reply:
Yes, we could. Either we create a dataset somewhere or we create a function that adds all the other stuff to the tokens so you only define what is needed for the test. I'll look into it.